<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1611247414575' stop='1611247505623'>
    <name>Invalid Credentials</name>
    <title>Invalid Credentials</title>
    <test-cases>
        <test-case start='1611247414576' status='broken' stop='1611247505622'>
            <name>"before all" hook for Invalid Credentials</name>
            <title>"before all" hook for Invalid Credentials</title>
            <labels>
                <label name='language' value='javascript'/>
                <label name='framework' value='wdio'/>
                <label name='thread' value='0-2'/>
            </labels>
            <parameters>
                <parameter kind='argument' name='browser' value='chrome-87.0.4280.141'/>
            </parameters>
            <steps>
                <step start='1611247414582' status='broken' stop='1611247505622'>
                    <name>POST /session/:sessionId/url</name>
                    <title>POST /session/:sessionId/url</title>
                    <attachments>
                        <attachment title='Request' source='f091be17-a5ac-492a-9b9e-a881c1d471be-attachment.json' type='application/json' size='31'/>
                    </attachments>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <failure>
                <message>Timeout of 90000ms exceeded. The execution in the test "Invalid Credentials "before all" hook for "should navigate to Sign In "" took too long. Try to reduce the run time or increase your timeout for test specs (https://webdriver.io/docs/timeouts.html). (/Users/user/skywebdriverio_test/test/specs/skylogin.spec.js)</message>
                <stack-trace>Error: Timeout of 90000ms exceeded. The execution in the test "Invalid Credentials "before all" hook for "should navigate to Sign In "" took too long. Try to reduce the run time or increase your timeout for test specs (https://webdriver.io/docs/timeouts.html). (/Users/user/skywebdriverio_test/test/specs/skylogin.spec.js)
                    at Hook.Runnable._timeoutError (/Users/user/skywebdriverio_test/node_modules/mocha/lib/runnable.js:432:10)
                    at Timeout.&lt;anonymous> (/Users/user/skywebdriverio_test/node_modules/mocha/lib/runnable.js:244:24)
                    at listOnTimeout (internal/timers.js:549:17)
                    at processTimers (internal/timers.js:492:7)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>